docs/core/integration-quality-scale/rules/iqs031.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "IQS031 - Icon translations"
+---
+
+IQS031 is as follows:
+
+Icon translations
+
+## Reasoning
+
+In the past, icons were part of the state of the integration.
+This was not really necessary, as they were usually either static or had a fixed set of states.
+
+To relief the state machine, icon translations were introduced.
+The name of this feature sounds weird since it is not about translating the icon itself, but rather referencing an icon by a translation key.
+The idea behind icon translations is that the integration defines icons in a file, which is then used by the frontend to display the icon.
+This also adds support for icons for the possible preset modes of a climate entity to pick from for example.
+
+Since adding icon translations to an integration betters the consistency between integrations, we consider it a best practice to use them.
+
```

Comment:
I don't think it has anything to do with consistency, besides following the IQS rules will improve consistency in general.
```suggestion
```